res/css/views/right_panel/_ThreadPanel.scss

```diff
@@ -119,17 +119,10 @@ limitations under the License.
         }
     }
 
+    // For style rules of EventTile in a thread, see _EventTile.scss
     &.mx_ThreadView {
         max-height: 100%;
 
-        // Inside a thread timeline only
-        .mx_GenericEventListSummary {
```

Comment:
Moved to https://github.com/matrix-org/matrix-react-sdk/pull/8689/files#diff-bd83194d78de871bc4ad1c4e8d43102c4368481bbb90705152d3d17fea813919R1029